### ISSUE TITLE
LMS's grading "out of" or "maximum" is ignored

### ISSUE THREAD

**Issue by seanh** [hypothesis/lms#1169]
In most LMS's, when you create or edit an assignment, you can set how many points the assignment should be graded out of. E.g. 0-3, 0-10, 0-100.

Our app ignores this setting and always asks you to enter a grade from 0 to 10.

This grade is displayed scaled in the LMS's gradebook. E.g. if the assignment is graded out of 100, and you entered 5 (out of 10) it will appear in our app as 5 but in the LMS as 50 or 50.0.

I think the number of decimal places to use is also a setting in LMS's that we ignore.

**Comment by jon-betts** [comment]
The operating theory is that we can't get these via LTI parameters, and that we might need to connect to the Moodle API (or another). We'd need to:

 * Confirm this isn't part of LTI
 * Confirm that this *is* accessible via the Moodle (or another) API

**Comment by mkdir-washington-edu** [comment]
I learned from our D2L consultant ([see this doc for details](https://docs.google.com/document/d/1p7XiZclKWDlkOxlcuHXdkrsGtemGeR5gQiJ4qUq7o00/edit?usp=sharing) that D2L is going to always assign a value of 100 points to a Hypothesis-generated grade item and users can pre-create the grade item before grading if that want to see different behavior.

> Great question, I am assuming your app syncs grades using LTI services. If so, then no its not possible to change the default value. It’s a limitation of the LTI spec itself, and not your our our fault. LTI limits that default value. What that actually looks like is your app can only send us, the LMS, a point value between 0.0-1.0. A grade sent to us for say an 80% would be sent over as a value of “0.8”. But the default LMS grade object will always have that default value of 100.
> 
> I would instead recommend that these customers pre-create the grade items ahead of time. Instead of relying LTI to auto-create the grade items for your activities, they should create the grade item and associate it to the LTI link manually. By doing so, they can create the grade to have any settings they want.

I've also seen similar behavior in Sakai and received an uncertain answerer from a Sakai partner when asking them if there was a way to change the default points for a Hypothesis Lessons item away from 100 points.

Bearing this in mind, I'm wondering if it might be better to always prompt users to give a grade as a percentage (out of 100) as opposed to trying to pull the points in from the LMS. Especially as the points for an assignment might not even be created until a grade is submitted in D2L and Sakai.

**Comment by mkdir-washington-edu** [comment]
I created a similar issue here: https://github.com/hypothesis/support/issues/187.

It looks like the LTI standard doesn't allow us to know the max grade set in the assignment. Instructors get confused because:
- They don't know we can't get the max grade, so they assume we've changed it to 10
- The don't know they can use decimals, so they don't think to leave a grade like 3.33/10 to get close to 1/3

A possible solution would be to ask for a percentage instead of points out of 10. This implies that we'll apply the percentage to the max grade, it avoids confusion over a max grade being shown, and gets us one more decimal place to work with.

After instructors enter something like "98%" we should do the math in the browser and send 9.8 to the LMS the way we currently do.
